spec/dataframe_spec.rb

```diff
@@ -943,6 +947,35 @@
     end
   end
 
+  context "#recode_date_time" do
```

Comment:
The contents of the `context` specify the method that is being tested. There is no method called `#recode_date_time`.

You should create `it` blocks inside the `context "#recode"` and add the tests there. Also, when you do add this to the `context "#recode"` block, try to write the `before { ... }` without repeating dataframes.

PS: The string passed to `it` should describe exactly what the test does. Read this guide on rspec: http://betterspecs.org/